contracts/lib/GatewayLib.sol

```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.5.0;
 
-// Copyright 2018 OpenST Ltd.
+// Copyright 2019 OpenST Ltd.
```

Comment:
Add a newline on line number 25.

Reply:
Why is `import "./RLPEncode.sol";` and `import "./SafeMath.sol";` needed in this library?